### PR TITLE
doc: stop generating PDF documentation

### PR DESCRIPTION
**Pull request by fabiobaltieri**
Stop generating a PDF version of the Zephyr documentation. This is a 3000+ pages document, the workflow that creates it has been notoriously problematic to the point that it's been disabled for pull request long time ago and it's now giving troubles running on the main target as well.

Stop generating the PDF document altogether and drop the references, we can always revisit this if it turns out that there was actual demand for it.

### CODE REVIEW

**Comment by fabiobaltieri** [comment]
@kartben do we have any data on how often that file has been downloaded? https://docs.zephyrproject.org/latest/zephyr.pdf

**Review by gmarull — APPROVED** [review]
+1, pdf is unusable

**Comment by kartben** [comment]
@fabiobaltieri it's downloaded more than I'd have anticipated (I'll try to pull the numbers but of the top of my head it's like 50 times a day, excluding "known" bots). I agree it's in a bad shape and probably not in a usable state, but oth I've been told it will be a required artifact in the context of safety certification.

I would certainly like to discuss this topic further next week during the documentation BoF, and would lean towards -1 until then.

File: doc/contribute/documentation/generation.rst
Comment (kartben) [inline doc/contribute/documentation/generation.rst:1]: there's more PDF-related stuff in this file that could be removed (if we agree to drop it), in particular I don't think LaTeX would be needed anymore as a dependency in the "Install the documentation processors section"

**Review by kartben — CHANGES_REQUESTED** [review]

**Review by nashif — CHANGES_REQUESTED** [review]
we need to be able to generate PDFs, whether this is useful as an alternative to html or whether it makes sense to generate as part of the process of building the docs and publishing it is a different question, but the infrastructure needs to work. 

Lets not remove something like that just when something fails for whatever reason. The discussion about removing/keepign this needs to happen first before we just remove it in a PR.

**Comment by fabiobaltieri** [comment]
> we need to be able to generate PDFs, whether this is useful as an alternative to html or whether it makes sense to generate as part of the process of building the docs and publishing it is a different question, but the infrastructure needs to work.
> 
> Lets not remove something like that just when something fails for whatever reason. The discussion about removing/keepign this needs to happen first before we just remove it in a PR.

Ack, it's just odd to have this middle ground where it's problematic so it's skipped in PR runs but we run it on push so one has to bisect to find what broke it. Guess we can revisit if it gives more troubles down the road.

**Comment by nashif** [comment]
> Ack, it's just odd to have this middle ground where it's problematic so it's skipped in PR runs but we run it on push so one has to bisect to find what broke it. Guess we can revisit if it gives more troubles down the road.

first thing we need to fix is that dependency of publishing the html pages on succesfull PDF build and the fact that it fails without anyone noticing (well, afew days later). This should just block somewhere so we can fix it.

**Pull request closed without merging**